src/lib/header-bidding/prebid/bid-config.spec.ts

```diff
@@ -540,11 +540,13 @@ describe('bids', () => {
 			[createAdSize(728, 90)],
 			mockPageTargeting,
 		)[2];
-		expect(openXBid.params).toEqual({
-			customParams: 'someAppNexusTargetingObject',
-			delDomain: 'guardian-d.openx.net',
-			unit: '540279541',
-		});
+		if (typeof openXBid !== 'undefined') {
+			expect(openXBid.params).toEqual({
+				customParams: 'someAppNexusTargetingObject',
+				delDomain: 'guardian-d.openx.net',
+				unit: '540279541',
+			});
+		}
```

Comment:
You could make this a touch more concise with optional chaining

```suggestion
	expect(openXBid?.params).toEqual({
		customParams: 'someAppNexusTargetingObject',
		delDomain: 'guardian-d.openx.net',
		unit: '540279541',
	});
```

Reply:
Yeah fair point! I initially tried this but it triggered the `no-unnecessary-condition` rule, and I preferred to minimise the number of times I had to disable eslint rules. But still did it anyway in another file 😅

Reply:
Yeah introducing `no-unnecessary-condition` errors once we turn the rule off is a tricky one actually